config/uitests-amo.js

```diff
@@ -0,0 +1,15 @@
+module.exports = {
+  apiHost: 'http://olympia.test',
+  proxyApiHost: 'http://olympia.test',
+  proxyPort: 3000,
+  proxyEnabled: true,
```

Comment:
I see that the `uitests-disco` config has a different setup, why?

Also, why do we have a `uitests` config and a `functional-development-amo`  config?

Reply:
```uitests-disco``` runs in a different environment and doesnt need fxa login. I should get rid of the other script as well.

Reply:
right but the api/proxy are not related to fxa, no?

Reply:
Not that I know of.

Reply:
does that mean `uitests-disco` relies on an API that is not reachable at `http://olympia.test`?